daft/context.py

```diff
@@ -329,7 +371,6 @@ def set_execution_config(
     shuffle_aggregation_default_partitions: int | None = None,
     read_sql_partition_size_bytes: int | None = None,
     enable_aqe: bool | None = None,
-    enable_native_executor: bool | None = None,
```

Comment:
should we leave this flag in since some users are using it already?

Reply:
Yup, will keep it in but also have a warning to tell them to switch to `DAFT_RUNNER=native`